cap-helm-charts-ci/tasks/create_pr.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This script is used to open a PR against this repository:
+# https://github.com/suse/kubernetes-charts-suse-com
+# It will fetch the bundle specified by CAP_BUNDLE and will update the helm
+# charts in the above repo.
+
+set -e
+
+if [ -z "${GIT_USER}"  ]; then
+  echo "GIT_USER environment variable not set"
+  exit 1
+fi
+
+if [ -z "${GIT_MAIL}"  ]; then
+  echo "GIT_MAIL environment variable not set"
+  exit 1
+fi
+
+if [ -z "${GITHUB_PRIVATE_KEY}" ]; then
+  echo "GITHUB_PRIVATE_KEY environment variable not set"
+  exit 1
+fi
+
+if [ -z "$GITHUB_TOKEN"  ]; then
+  echo "GITHUB_TOKEN environment variable not set"
+  exit 1
+fi
+
+if [ -z "${CHART_BUNDLE}" ]; then
+  echo "CHART_BUNDLE environment variable not set"
+  exit 1
+fi
+
+if [ -z "${HELM_CHARTS_REPO}" ]; then
+  echo "HELM_CHARTS_REPO environment variable not set"
+  exit 1
+fi
+
+# Setup git
+mkdir -p ~/.ssh
+echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" >> ~/.ssh/known_hosts
```

Comment:
This is not safe - is `ssh-keyscan` an option?

Reply:
DONE.